src/woo_publications/api/drf_spectacular/extensions.py

```diff
@@ -0,0 +1,22 @@
+from drf_spectacular.contrib.django_filters import DjangoFilterExtension
+
+from woo_publications.api.utils import underscore_to_camel
+
+
+class CamelizeFilterExtension(DjangoFilterExtension):
+    priority = 1
+
+    def get_schema_operation_parameters(self, auto_schema, *args, **kwargs):
+        """
+        camelize query parameters
+        """
+        parameters = super().get_schema_operation_parameters(  # pragma: no cover
+            auto_schema, *args, **kwargs
+        )
+
+        for parameter in parameters:
+            parameter["name"] = underscore_to_camel(
+                parameter["name"]
+            )  # pragma: no cover
```

Comment:
See the annotation - you need to put the comment on line 17 or 18 for it to take effect (black formatting tends to trip these up)